fdbmonitor/CMakeLists.txt

```
@@ -1,6 +1,7 @@
 set(FDBMONITOR_SRCS ConvertUTF.h SimpleIni.h fdbmonitor.cpp)
 
 add_executable(fdbmonitor ${FDBMONITOR_SRCS})
+target_link_libraries(fdbmonitor rt)
```

```suggestion
if(UNIX AND NOT APPLE)
    target_link_libraries(fdbmonitor rt)
endif()
```
This won't work under Windows and it is probably unnecessary for OS X.